sphinx/source/models.rst

```diff
@@ -182,4 +182,4 @@ Inducing Point Allocators
 Other Utilties
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .. automodule:: botorch.models.utils.assorted
-    :members:
+    :members:
```

Comment:
what happened here?

Reply:
Not sure what happened. I removed `SingleTaskAugmentedGP` from the doc. Community contributions shouldn't go in the documentation, right?

Reply:
@saitcakmak what are your thoughts re community contribs? Would be nice if we generated docs as part of the community modules (maybe not part of the core modules) given that people are putting effort into their docstrings (thanks!).

Reply:
Yeah, I think it'd be good to have generated docs for community modules on the website. We can build it when building the community tutorials for the website.